src/components/PackageCard/Heading.tsx

```diff
@@ -10,14 +10,15 @@ import { Link } from "react-router-dom";
 import { useLanguage } from "../../hooks/useLanguage";
 import { getPackagePath } from "../../util/url";
 import { CDKTypeIcon, CDKTypeText } from "../CDKType";
+import { EditorsNote } from "./EditorsNote";
 import { usePackageCard } from "./PackageCard";
 import testIds from "./testIds";
 
 export const Heading: FunctionComponent = () => {
```

Comment:
This isn't part of your changes, but I did want to raise the question of whether we should change the card title to use `wordBreak="normal"` instead of `wordBreak="break-all"`, so that it prefers to break on dashes instead of in the middle of words. I feel like this could improve the readability in some areas, but I wonder if you or anyone else has found any edge cases where its problematic.

Reply:
Good callout, updated 👍